packages/SketchMorph2-Core.package/M2MenuItemMorph.class/methodProperties.json

```diff
@@ -10,5 +10,4 @@
 		"isDefaultTool" : "mFr 5/6/2021 15:56",
 		"isDefaultTool:" : "mFr 5/6/2021 15:53",
 		"isM2MenuItemMorph" : "mFr 5/19/2021 09:04",
-		"isSelected:" : "ms 5/13/2021 19:21",
```

Comment:
Any reason why this was removed? (same as in #61 and #59)

Reply:
See my answer in #59